bin/setup

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+set -e
+
+# Set up Ruby dependencies via Bundler
+gem install bundler --conservative
+bundle check || bundle install
+
+# Add binstubs to PATH via export PATH=".git/safe/../../bin:$PATH" in ~/.zshenv
+mkdir -p .git/safe
+
+if ! command -v bower > /dev/null; then
+  npm install -g bower
+fi
+
+if ! [ -d spec/dummy/my-app ]; then
+  git clone https://github.com/kellyselden/ember-cli-output.git spec/dummy/my-app
+fi
+
+root="$(pwd)"
+
+cd ${root}/spec/dummy/my-app &&
```

Comment:
I tried that:

```
bin/setup: line 24: cd: /Users/seanpdoyle/src/ember-cli-rails/spec/dummy/my-app/spec/dummy: No such file or directory
```

I think it's important to capture the current directory for the sake of the `cd` commands, right?